magento/pwa-studio
# [PWA-808] Allow PWA Studio to use a different store to localize CMS Pages

## Description
Add ability to localize CMS Pages using the `STORE_VIEW_CODE` env variable

## Related Issue
https://jira.corp.magento.com/browse/PWA-808

## Acceptance 

### Verification Stakeholders
@jimbo 
@anthoula 
@eug123 
@dpatil-magento 

### Specification
- Developer is able to define the desired store for the application to use
- CMS Page data should be retrieve from the store

### Verification Steps
- Create a CMS Page in the Magento admin associated to a specific store view
- Modify the `STORE_VIEW_CODE` to point to that store
- Verify the CMS Page is loaded from the correct store
- Create a pair of CMS Pages with the same URL key between store views
- Modify `STORE_VIEW_CODE` to both store views and verify store changes after a restart of the `watch` command

## Screenshots / Screen Captures (if appropriate)

## Checklist
